src/Microsoft.AspNetCore.SystemWebAdapters/HttpException.cs

```diff
@@ -1,10 +1,15 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System;
+using System.Net;
+
 namespace System.Web;
 
 public class HttpException : SystemException
 {
+    private readonly int httpStatusCode = 500;
```

Comment:
we've been using `_` as prefix for fields. Can you update to that and remove the `this.`? I'll see about getting a style enforcement here for future commits

Reply:
Fixed, thanks!